app/views/home/index.html.erb

```diff
@@ -9,3 +9,15 @@
   <li><%= link_to category.name, category_path(category.id) %></li>
 <% end %>
 </ul>
+
+<% if current_user %>
+  <div>
+    <h4>You are logged in with <%= current_user.email %></h4>
+    <%= link_to t("auth.logout"), logout_path %>
+  </div>
+<% else %>
+  <div>
+    <%= link_to t("auth.login"), login_path %> or
+    <%= link_to t("auth.create_account"), new_user_path %>
+  </div>
+<% end %>
```

Comment:
This feels like a good candidate for a partial. What do you think?

Reply:
Good idea!  I'm not sure how I'm going to show the logged in state in actuality, but when I have figured that out, I'll make sure to make the code a partial :)